python/testing/__init__.py

```diff
@@ -215,6 +196,99 @@ def assertFilesEqual(self, filepath_expected, filepath_result):
                 diff = list(diff)
                 self.assertEqual(0, len(diff), ''.join(diff))
 
+    def checkGeometriesEqual(self, geom0, geom1, geom0_id, geom1_id, use_asserts=False, precision=14, topo_equal_check=False):
+        """ Checks whether two geometries are the same - using either a strict check of coordinates (up to given precision)
+        or by using topological equality (where e.g. a polygon with clockwise is equal to a polygon with counter-clockwise
+        order of vertices) """
```

Comment:
Can we also add a `.. versionadded:: 3.2` so python plugins doing CI can be aware of these new functions?
Same for `checkAttributesEqual`